docker-compose-collector.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+services:
+  php:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    volumes:
+    - ./:/usr/src/myapp
+    depends_on:
+      - otel-collector
+  zipkin:
+    image: openzipkin/zipkin-slim
+    ports:
+    - 9411:9411
+  jaeger:
+    image: jaegertracing/all-in-one
+    environment:
+      COLLECTOR_ZIPKIN_HTTP_PORT: 9412
+    ports:
+    - 9412:9412
+    - 16686:16686
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib
+    command: ["--config=/etc/otel-collector-config.yml"]
+    volumes:
+      - ./files/collector/otel-collector-config.yml:/etc/otel-collector-config.yml
+    ports:
+      - "1888:1888"   # pprof extension
+      - "8888:8888"   # Prometheus metrics exposed by the collector
+      - "8889:8889"   # Prometheus exporter metrics
+      - "13133:13133" # health_check extension
+      - "9411"   # Zipkin receiver
+      - "4317:4317"        # OTLP gRPC receiver
+      - "55680:55679" # zpages extension
```

Comment:
The comments might need a bit of formatting here but that is a tiny nit that you can totally ignore!